package.json

```diff
@@ -69,5 +69,8 @@
     "tar-stream": "^2.1.0",
     "which": "^1.2.0",
     "yauzl": "^2.10.0"
+  },
+  "dependencies": {
+    "bpmn-js": "^9.0.3"
```

Comment:
Was this added by accident?

Reply:
Yes, not sure what happened there. I'll remove